### PR TITLE
Update 12 raw data analysis

### PR DESCRIPTION
**Pull request by dschust-r**
Added QC section with software table

Please provide a description of your changes:

Please check the following about your suggested changed:
- [x] The new text has one sentence per line
- [x] preprints that are cited are listed as an issue for discussion and peer review

The only preprint cited is the AlphaPept preprint (10.1101/2021.07.23.453379), how exactly would you like to discuss this one? Should I post an issue on it or mention in the table that it is a preprint?

### CODE REVIEW

**Comment by jessegmeyerlab** [comment]
Thanks for contributing more! I should be able to review this over the weekend. 

For preprints we just want to make sure we vet their claims since they aren't peer reviewed. There is a template for new paper discussion if you click the new issue button. Since it's software you don't have to be to harsh in the review - either it does what they say or it doesn't. 

Thanks again for contributing!!

**Comment by dschust-r** [comment]
Edit: Posted an issue about the AlphaPept preprint

File: content/12.raw-data-analysis.md [hunk 1]
```diff
@@ -107,6 +107,74 @@ $$FDR = \frac{2 * Decoy PSMs}{Target + Decoy PSMs}$$
...
+Quality control should be a central aspect of any mass spectrometry-based study to ensure reproducibility of generated results.
+There are two types of quality controls that can be conducted for any kind of mass spectrometry experiment.
+The first one is focused on monitoring the perfomance of the instruments themselves (e.g. HPLC and mass spectrometer), whereas the second one is focused on your experiments. 
+For further reading, we recommend to take a look at issue 11 on quality control published in the journal _Proteomics_ in 2011 [@DOI:10.1002/pmic.201190020], especially the review by Köcher _et al._ [@DOI:10.1002/pmic.201000578], as well as the review published by Bittremieux _et al._ in 2017 [@DOI:10.1002/mas.21544].
+
+#### Instrument Performance
+
+It is generally advisable to monitor instrument performance regularly. 
+Instrument calibrations in regular intervals help ensure that performance is appropriate.
+During the calibration you can check injection times and intensity of the ions of your calibration mix.
```
Comment (jessegmeyerlab) [inline content/12.raw-data-analysis.md]:
```suggestion
During the calibration you can check injection times (for ion trap instruments) and intensity of the ions in the calibration mix.
```

File: content/12.raw-data-analysis.md [hunk 2]
```diff
@@ -107,6 +107,74 @@ $$FDR = \frac{2 * Decoy PSMs}{Target + Decoy PSMs}$$
...
 
+Quality control should be a central aspect of any mass spectrometry-based study to ensure reproducibility of generated results.
+There are two types of quality controls that can be conducted for any kind of mass spectrometry experiment.
+The first one is focused on monitoring the perfomance of the instruments themselves (e.g. HPLC and mass spectrometer), whereas the second one is focused on your experiments. 
+For further reading, we recommend to take a look at issue 11 on quality control published in the journal _Proteomics_ in 2011 [@DOI:10.1002/pmic.201190020], especially the review by Köcher _et al._ [@DOI:10.1002/pmic.201000578], as well as the review published by Bittremieux _et al._ in 2017 [@DOI:10.1002/mas.21544].
+
+#### Instrument Performance
+
+It is generally advisable to monitor instrument performance regularly. 
+Instrument calibrations in regular intervals help ensure that performance is appropriate.
```
Comment (jessegmeyerlab) [inline content/12.raw-data-analysis.md]:
```suggestion
Instrument calibrations in regular intervals help ensure that performance is maintained.
Often basic calibration and sensitivity can be checked by direct infusion of a standard.
```

File: content/12.raw-data-analysis.md [hunk 3]
```diff
@@ -107,6 +107,74 @@ $$FDR = \frac{2 * Decoy PSMs}{Target + Decoy PSMs}$$
...
+During the calibration you can check injection times and intensity of the ions of your calibration mix.
+To find out if your mass spectrometer is performing as expected, you can analyze complex samples, such as tryptic digests of whole-cell lysates (e.g. HeLa cells, HEK cells, yeast, etc.) or tryptic digests of purified proteins.
```
Comment (jessegmeyerlab) [inline content/12.raw-data-analysis.md]:
```suggestion

After ensuring good calibration and signal with the simple calibration mixture, it is advisable to analyze complex samples, such as tryptic digests of whole-cell lysates (e.g. HeLa cells, HEK cells, yeast, etc.) or tryptic digests of purified proteins.
The additional check with a complex sample ensures all aspects of the system are working together correctly, especially the liquid chromatography and emitter.
```

File: content/12.raw-data-analysis.md [hunk 4]
```diff
@@ -107,6 +107,74 @@ $$FDR = \frac{2 * Decoy PSMs}{Target + Decoy PSMs}$$
...
+These digests should be analyzed after every instrument calibration and if required in between samples when acquiring more extensive batches.
```
Comment (jessegmeyerlab) [inline content/12.raw-data-analysis.md]:
```suggestion
These digests should be analyzed after every instrument calibration and periodically between samples when acquiring more extensive batches.
```

File: content/12.raw-data-analysis.md [hunk 5]
```diff
@@ -107,6 +107,74 @@ $$FDR = \frac{2 * Decoy PSMs}{Target + Decoy PSMs}$$
...
+Measured tryptic digests can be searched with software of your choice and the numbers of identified peptide precursors and proteins can be compared.
```
Comment (jessegmeyerlab) [inline content/12.raw-data-analysis.md]:
```suggestion
Data measured from tryptic digests should be analyzed by the software of your choice and the numbers of identified peptide precursors and proteins can be compared with previous controls for consistency.
```

File: content/12.raw-data-analysis.md [hunk 6]
```diff
@@ -107,6 +107,74 @@ $$FDR = \frac{2 * Decoy PSMs}{Target + Decoy PSMs}$$
...
+Measurments of digested purified proteins are great for identifying retention time shifts and mass accuracy problems.
```
Comment (jessegmeyerlab) [inline content/12.raw-data-analysis.md]:
```suggestion

Another strategy is to analyze digested purified proteins, which easily enable discovery of retention time shifts and mass accuracy problems. 
```

File: content/12.raw-data-analysis.md [hunk 7]
```diff
@@ -107,6 +107,74 @@ $$FDR = \frac{2 * Decoy PSMs}{Target + Decoy PSMs}$$
...
+During the calibration you can check injection times and intensity of the ions of your calibration mix.
+To find out if your mass spectrometer is performing as expected, you can analyze complex samples, such as tryptic digests of whole-cell lysates (e.g. HeLa cells, HEK cells, yeast, etc.) or tryptic digests of purified proteins.
+These digests should be analyzed after every instrument calibration and if required in between samples when acquiring more extensive batches.
+Measured tryptic digests can be searched with software of your choice and the numbers of identified peptide precursors and proteins can be compared.
+Measurments of digested purified proteins are great for identifying retention time shifts and mass accuracy problems. 
+In case you are working with a Thermo mass spectrometer, you can open the acquired .raw file directly either in FreeStyle or in Qual Browser and look for specific m/z values of your peptides.
+Looking at the intensity of the extracted peaks will help identify sensitivity fluctuations. 
+Carry-over between different measurments can be identified from blank measurements which are subsequently analyzed with your search software of choice.
```
Comment (jessegmeyerlab) [inline content/12.raw-data-analysis.md:131]:
```suggestion

Carry-over between different measurments can be identified from blank measurements which are subsequently analyzed with your search software of choice.
```

File: content/12.raw-data-analysis.md [hunk 8]
```diff
@@ -107,6 +107,74 @@ $$FDR = \frac{2 * Decoy PSMs}{Target + Decoy PSMs}$$
...
+Blank measurements can be injections of different buffers, water or the starting conditions of your liquid chromatography. 
+In case of increased detection of carry-over, injections with trifluoroethanol can be performed.
+Another factor to take into consideration is the stability of your electrospray.
```
Comment (jessegmeyerlab) [inline content/12.raw-data-analysis.md:135]:
```suggestion

Another factor to take into consideration is the stability of your electrospray.
```

File: content/12.raw-data-analysis.md [hunk 9]
```diff
@@ -107,6 +107,74 @@ $$FDR = \frac{2 * Decoy PSMs}{Target + Decoy PSMs}$$
...
+Electrospray stability tends to worsen over time with column wear-out, as well as when measuring samples with residual contaminants, such as salts or detergents.
```
Comment (jessegmeyerlab) [inline content/12.raw-data-analysis.md]:
```suggestion
Electrospray stability tends to worsen over time as columns wear, as well as when measuring samples with residual contaminants, such as salts or detergents. 
```

**Review by jessegmeyerlab — CHANGES_REQUESTED** [review]
this is great overall, thank you Dina! most suggestions are to break into paragraphs. Let me know what you think

**Comment by dschust-r** [comment]
Thank you for reviewing and suggesting changes! 
The suggestions all looked sensible to me, I committed them to the branch.

**Pull request merged**